.github/workflows/new-pull-request.yaml

```diff
@@ -4,6 +4,9 @@ on:
   pull_request_target:
     branches: [main]
     types: [opened, reopened, synchronize]
+    paths-ignore: 
+     - 'Validated_Partners/**'
+     - '.github/**'
```

Comment:
What is this? Why are we doing this ignore?

Reply:
This ignore is to stop the run from triggering on PR's where the only files changed came from one of these two paths  GH actions updates or the automated Validated_Partners runs. If other files not found in these folders are changed in a PR the action will still run